### ISSUE TITLE
Node.js Foundation Technical Steering Committee (TSC) Meeting 2019-09-04

### ISSUE THREAD

**Issue by mhdawson** [nodejs/TSC#752]
## Time

**UTC Wed 04-Sep-2019 19:00 (07:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 04-Sep-2019 12:00 (12:00 PM) |
| US / Mountain | Wed 04-Sep-2019 13:00 (01:00 PM) |
| US / Central  | Wed 04-Sep-2019 14:00 (02:00 PM) |
| US / Eastern  | Wed 04-Sep-2019 15:00 (03:00 PM) |
| London        | Wed 04-Sep-2019 20:00 (08:00 PM) |
| Amsterdam     | Wed 04-Sep-2019 21:00 (09:00 PM) |
| Moscow        | Wed 04-Sep-2019 22:00 (10:00 PM) |
| Chennai       | Thu 05-Sep-2019 00:30 (12:30 AM) |
| Hangzhou      | Thu 05-Sep-2019 03:00 (03:00 AM) |
| Tokyo         | Thu 05-Sep-2019 04:00 (04:00 AM) |
| Sydney        | Thu 05-Sep-2019 05:00 (05:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Technical%20Steering%20Committee%20(TSC)+Meeting+2019-09-04&iso=20190904T19
* or http://www.wolframalpha.com/input/?i=07PM+UTC%2C+Sep+04%2C+2019+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1uQ_9yM-NiB7FMYoG7Sju6-j2vIHdnkcDoR4qg2XBksU/edit>

## Agenda

Extracted from **tsc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* stream: don't emit error after destroy() [#29197](https://github.com/nodejs/node/pull/29197)
* build: ongoing list of actions for Python 3 compatibility [#25789](https://github.com/nodejs/node/issues/25789)
* http2: make compat finished match http/1 [#24347](https://github.com/nodejs/node/pull/24347)

### nodejs/TSC

* Nominating @tniessen for the TSC [#746](https://github.com/nodejs/TSC/issues/746)
* Nominating @BethGriggs to the TSC [#718](https://github.com/nodejs/TSC/issues/718)

### nodejs/admin

* Moderation team recertification [#408](https://github.com/nodejs/admin/issues/408)

## Invited

* Anna Henningsen @addaleax (TSC)
* Anatoli Papirovski @apapirovski (TSC)
* Сковорода Никита Андреевич @ChALkeR (TSC)
* Colin Ihrig @cjihrig (TSC)
* Daniel Bevenius @danbev (TSC)
* Franziska Hinkelmann @fhinkel (TSC)
* Jeremiah Senkpiel @Fishrock123 (TSC)
* Gabriel Schulhof @gabrielschulhof (TSC)
* Gireesh Punathil @gireeshpunathil (TSC)
* James Snell @jasnell (TSC)
* Joyee Cheung @joyeecheung (TSC)
* Matteo Collina @mcollina (TSC)
* Michael Dawson @mhdawson (TSC)
* Myles Borins @MylesBorins (TSC)
* Sam Roberts @sam-github (TSC)
* Michaël Zasso @targos (TSC)
* Sakthipriyan Vairamani @thefourtheye (TSC)
* Rich Trott @Trott (TSC)

### Observers/Guests
Beth Griggs @BethGriggs (guest participant)
Christian Clauss @cclauss (guest participant)
Tobias Nießen @tniessen (guest participant)

## Notes

The agenda comes from issues labelled with `tsc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Zoom link: https://zoom.us/j/611357642
Regular password

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the TSC that's not worth putting on as a separate agenda item, this is a good place for that.

---

**Invitees**

Please use the following emoji reactions in this post to indicate your
availability.

* :+1: - Attending
* :-1: - Not attending
* :confused: - Not sure yet

**Comment by Trott** [comment]
TL;DR: Hey, @bnb, @dshaw, @gdams, @gibfahn, @hackygolucky, @ljharb, @thefourtheye: can you folks indicate whether you are active in Moderation these days and wish to remain on the team? Whatever your answer, thank you for doing the hard Moderation Team work this year!

> Moderation team recertification #408

Regarding that item: Current Moderation Team members are:

* @benjamingr
* @bnb
* @dshaw
* @gdams
* @gibfahn
* @hackygolucky
* @ljharb
* @othiym23
* @ryanmurakami
* @thefourtheye

Of those team members, @benjamingr and @ryanmurakami are active based on activity in the moderation repository. And @othiym23 [chimed in on behalf of Moderation Team](https://github.com/nodejs/TSC/issues/744#issuecomment-521384866) not that long ago.

AFAICT, Moderation Team has not had a public meeting for a long time and I don't know where their minutes are kept. I suspect most of the conversations happen in the private Slack channel. So, rather than try to dig through a public record that I don't know where to find, I'm just flat-out asking folks if they've been doing stuff.

**Comment by ljharb** [comment]
I’m still active :-) luckily there’s not been much that requires activity.

**Comment by ChALkeR** [comment]
I labelled https://github.com/nodejs/TSC/issues/748 with `tsc-agenda`, I think that we should just move 19 UTC timeslot to 20 UTC.

**Comment by mhdawson** [comment]
@ChALkeR added to agenda in the minutes doc.

**Comment by mhdawson** [comment]
We ended up cancelling.  8 mins after and there were only four of us.  Nothing urgent on the agenda so we'll talk about the issues next week.
